spec/fixtures/data_infer_row_limit.csv

```diff
@@ -1,5 +1,5 @@
 id,age,name
-1,39,Paul
```

Comment:
This change was needed to avoid having `name` inferred as `binary`. Apparently  `Paul` is a valid binary string and gets decoded to `=\xAB\xA5` :smiley: